pkg/subarcs/Library/SubarcsApp.js

```diff
@@ -0,0 +1,113 @@
+/**
+ * @license
+ * Copyright 2022 Google LLC
+ *
+ * Use of this source code is governed by a BSD-style
+ * license that can be found in the LICENSE file or at
+ * https://developers.google.com/open-source/licenses/bsd
+ */
+import '../conf/config.js';
+import '../../Library/App/surface-imports.js';
+import {logFactory, makeName} from '../../Library/Core/utils.min.js';
+import {DeviceUxRecipe} from '../../Library/Media/DeviceUxRecipe.js';
+import {LocalStoragePersistor} from '../../Library/LocalStorage/LocalStoragePersistor.js';
+import {App} from '../../Library/App/Worker/App.js';
+
+const log = logFactory(true, 'SubarcsApp', 'navy', 'silver');
+
+const SubarcsRecipe = {
```

Comment:
interesting! i can't quite figure out what this is, but i'm anticipatorily excited

Reply:
Experimental, will explain f2f.